lib/pure/httpcore.nim

```diff
@@ -17,6 +17,7 @@ import tables, strutils, parseutils
 type
   HttpHeaders* = ref object
     table*: TableRef[string, seq[string]]
+    convert: proc (str: string): string
```

Comment:
I don't think we really need this, it just adds extra complexity. Just store `isTitleCase`.

Reply:
I changed this accordingly.